custom_components/tuya_local/devices/trotec_pac-w_heatpump.yaml

```diff
@@ -0,0 +1,80 @@
+name: Air conditioner
+products:
+  - id: 6r9as02gkiaaxelt
+    name: Trotec PAC-W 2600 SH
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: COOL
+              value: cool
+            - dps_val: DRY
+              value: dry
+            - dps_val: FAN
+              value: fan_only
+            - dps_val: HEAT
+              value: heat
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 16
+        max: 30
+      unit: C
+    - id: 3
+      type: integer
+      name: current_temperature
+    - id: 4
+      type: string
+      name: mode
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: integer
+      mapping:
+        - dps_val: 1
+          value: low
+        - dps_val: 2
+          value: medium
+        - dps_val: 3
+          value: high
+    - id: 19
+      name: temperature_unit
+      type: string
```

Comment:
Are the values "C" and "F" (upper case)?  If they are lower case, as is more common, then a mapping will be needed.

It would also be useful to expose this as a select entity so the units can be changed from HA.

Reply:
Yes looks like upper case "C" and "F"

Reply:
I don't think it is really meant to be changed.
It is changeable via API, but not via tuya app (at least I can't find it) and the app does change the values but not the unit.